payment_method_object.go

```diff
@@ -0,0 +1,51 @@
+package easypost
```

Comment:
Issue: no need for an entirely new file, just drop these into the `payment_method` file. Go the language doesn't care at all about files, it's more for us. No need to fragment the code in multiple places.

Reply:
Resolved